src/test/java/seedu/task/commons/util/UrlUtilTest.java

```diff
@@ -1,14 +1,12 @@
 package seedu.task.commons.util;
 
```

Comment:
yea, we can delete it haha, but I think we can just say that these classes are there in case we want to do more things in the future. It adds to the test code that we write also. =)